staging/src/k8s.io/component-helpers/scheduling/corev1/helpers_test.go

```diff
@@ -545,123 +545,3 @@ func TestMatchNodeSelectorTermsStateless(t *testing.T) {
 		})
 	}
 }
-
-func TestTolerationsTolerateTaintsWithFilter(t *testing.T) {
```

Comment:
You could still run the test but with FindMatchingUntoleratedTaint in it.

Reply:
Oh right, thanks